src/content/plugins/split-chunks-plugin.md

```diff
@@ -0,0 +1,220 @@
+
+webpack will automatically split chunks based on these conditions:
+
+* New chunk can be shared OR modules are from the `node_modules` folder
+* New chunk would be bigger than 30kb (before min+gz)
+* Maximum number of parallel request when loading chunks on demand would be lower or equal to 5
+* Maximum number of parallel request at initial page load would be lower or equal to 3
+
+When trying to fullfill the last two conditions, bigger chunks are preferred.
+
+Let's take a look at some examples.
+
+### Example 1
+
+``` js
+// index.js
+
+// dynamically import a.js
+import("./a");
+```
+
+``` js
+// a.js
+import "react";
+
+// ...
+```
+
+**Result:** A separate chunk would be created containing `react`. At the import call this chunk is loaded in parallel to the original chunk containing `./a`.
+
+Why:
+
+* Condition 1: The chunk contains modules from `node_modules`
+* Condition 2: `react` is bigger than 30kb
+* Condition 3: Number of parallel requests at the import call is 2
+* Condition 4: Doesn't affect request at initial page load
+
+What's the reasoning behind this? `react` probably won't change as often as your application code. By moving it into a separate chunk this chunk can be cached separately from your app code (assuming you are using chunkhash, records, Cache-Control or other long term cache approach).
+
+### Example 2
+
+``` js
+// entry.js
+
+// dynamically import a.js and b.js
+import("./a");
+import("./b");
+```
+
+``` js
+// a.js
+import "./helpers"; // helpers is 40kb in size
+
+// ...
+```
+
+``` js
+// b.js
+import "./helpers";
+import "./more-helpers"; // more-helpers is also 40kb in size
+
+// ...
+```
+
+**Result:** A separate chunk would be created containing `./helpers` and all dependencies of it. At the import calls this chunk is loaded in parallel to the original chunks.
+
+Why:
+
+* Condition 1: The chunk is shared between both import calls
+* Condition 2: `helpers` is bigger than 30kb
+* Condition 3: Number of parallel requests at the import calls is 2
+* Condition 4: Doesn't affect request at initial page load
+
+Putting the content of `helpers` into each chunk will result into its code being downloaded twice. By using a separate chunk this will only happen once. We pay the cost of an additional request, which could be considered a tradeoff. That's why there is a minimum size of 30kb.
+
+T> With the new `optimizations.splitChunks.chunks: "all"` option the same would happend for initial chunks. Chunks can even be shared between entrypoints and on-demand loading.
+
+## Configuration
+
+For these people that like to have more control over this functionality, webpack provides a set of options to better fit your needs.
+
+W> If you are manually changing the split configuration, measure the impact of the changes to see and make sure there's a real benefit. The defaults are choosen to fit best practices of web performance.
```

Comment:
s/choosen/chosen
(change below my opinion, not related to grammar):
s/fit best practices of web performance/fit web performance best practices